bindings/graphql/graphql.go

```diff
@@ -0,0 +1,169 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package graphql
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/kit/logger"
+	graphql "github.com/machinebox/graphql"
```

Comment:
This library is not as active as this other one: https://github.com/graphql-go/graphql

Any reason why this one was chosen?

Reply:
@artursouza That is a very light library that is sufficient for building a binding.

Reply:
I like this choice because this library focuses on being a very simple GraphQL client and does not provide all the extra server-side support.  Also, main contributor is a well-known Gopher ;-)

If there are issues in the future w/r/t maintenance we can swap it out.

Reply:
@artursouza @yaron2 Can we look into this ? I think we would like this component in next release if possible.. We can always release the updated version if the library causes any issues ..We have been using it for a while and it is stable....

Reply:
@artursouza @yaron2 Can we look into this ? I think we would like this component in next release if possible.. We can always release the updated version if the library causes any issues ..We have been using it for a while and it is stable....

Reply:
OK. LGTM.